dictionary/Глаголы/Ь.txt

```diff
@@ -7473,17 +7473,17 @@
 подребезжа́ть св нп 5b
 продребезжа́ть св нп 5b
 переезжа́ть нсв 1a ◑перее́хать
-приезжа́ть нсв нп 1a ◑п́рие́хать
-доезжа́ть нсв нп 1a ◑д́ое́хать
-1/проезжа́ть нсв 1a ◑п́рое́хать
+приезжа́ть нсв нп 1a ◑прие́хать
+доезжа́ть нсв нп 1a ◑дое́хать
+1/проезжа́ть нсв 1a ◑прое́хать
```

Comment:
These, along with some others, were fixed in your previous pull request. You need to pull master into this branch or start a new branch from master. Otherwise we will keep seeing your old changes which is not ideal.

Reply:
If it helps, I've given you write access to this repo.

Reply:
Sorry for that. Actually this is the first time I've collaboratively contributed to a GitHub project, and I thank you very much for telling me the right way to do that. I'd better be more experienced with GitHub working flow before being given write access, lest I mess it up.
I'm supposed to pull gramdit's master into my patch-2 and start over, right?

Reply:
No problem at all. First, try [syncing](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/syncing-a-fork) your fork with gramdict's master branch. This should bring all changes made in gramdict/zalizniak-2010 master into your patch-2, leaving only the new changes you have made.

If syncing doesn't work for some reason, you can always start a new branch (patch-3) as described [here](https://github.com/gramdict/zalizniak-2010#%D0%BA%D0%B0%D0%BA-%D1%81%D0%BE%D0%B7%D0%B4%D0%B0%D1%82%D1%8C-pull-request) (assuming you can read Russian).

Reply:
Got it. Many thanks!